crypto/src/key/rschnorr/mod.rs

```diff
@@ -16,14 +16,12 @@
 mod internal;
```

Comment:
There's probably a misunderstanding on what this module is. This is just a portal to the tari crypto library. This has to go if you get rid of tari_crypto

Reply:
Understood, I have removed the unnecessary tests.